modules/release-notes/ref-release-notes-fixed-security-issues.adoc

```diff
@@ -2,4 +2,4 @@
 [id="fixed-security-issues"]
 = Fixed security issues
 
-You can view the security issues fixed in {product} {product-version} at link:https://access.redhat.com/security/security-updates/cve?q=red+hat+developer+hub&p=1&sort=cve_publicDate+desc,allTitle+desc&rows=10&documentKind=Cve[Red Hat Security Updates].
\ No newline at end of file
+You can view the security issues fixed in {product} 1.5.0 and {product-bundle-version} at link:https://access.redhat.com/errata/RHSA-2025:3374[Red Hat Security Advisory RHSA-2025:3374].
```

Comment:
A reminder that if you use variables everywhere, you need to later hardcode them when 1.5.2 comes out. 

In general for things that are versioned to NEVER MOVE, I would strongly recommend NOT using attributes that change with every .z update.